arcgis10_mapping_tools/MapAction/MapAction/Utilities.cs

```diff
@@ -103,6 +104,37 @@ public static string createXML(Dictionary<string, string> usDict, string rootEle
 
         #endregion
 
+        #region Public method createXML
+        //Creates an xml given a dictionary of tags and values.  Also pass in the root element, file path and filename.
+        public static string createXML(MapAction.OperationConfig operationConfig, string path, string fileName)
+        {
+            //set output path and filename
+            string pathFileName;
+            //check to see if the path is the root of the directory, if so remove additional \
+            string root = System.IO.Path.GetPathRoot(path);
+            if (root != path)
+            {
+                pathFileName = @path + "\\" + fileName + ".xml";
+                Debug.WriteLine(pathFileName);
+            }
+            else
+            {
+                pathFileName = @path + fileName + ".xml";
+                Debug.WriteLine(pathFileName);
+            }
+
+            using (StringWriter stringwriter = new System.IO.StringWriter())

```

Comment:
is this right? stringwriter doesn't seem to get used in the scope of the using block.